### PR TITLE
Fix due to #1027

### PR DESCRIPTION
**Pull request by Mortimal**
https://github.com/OpenKore/openkore/issues/1027

### CODE REVIEW

File: src/Commands.pm [hunk 1]
```diff
@@ -4633,6 +4633,12 @@ sub cmdStorage {
 			cmdStorage_desc($items);
 		} elsif ($switch eq 'add' || $switch eq 'addfromcart' || $switch eq 'get' || $switch eq 'gettocart' || $switch eq 'close') {
 			if ($char->storage->isReady()) {
+			if (!$items
```
Comment (allanon) [inline src/Commands.pm]: This works, but it's ugly. The `if` is spread over multiple lines, has unnecessary parentheses (some people swear by unnecessary parentheses (and I tend to use them when writing sentences too) so maybe that's not bad), and the indentation is wrong.

Also, it doesn't tell people why nothing happened. It's better to fall through to the usage if the syntax is wrong.

How about this instead?
```
diff --git a/src/Commands.pm b/src/Commands.pm
index ba7e6042f..e248e9492 100644
--- a/src/Commands.pm
+++ b/src/Commands.pm
@@ -4631,7 +4631,7 @@ sub cmdStorage {
                        cmdStorage_log();
                } elsif ($switch eq 'desc') {
                        cmdStorage_desc($items);
-               } elsif ($switch eq 'add' || $switch eq 'addfromcart' || $switch eq 'get' || $switch eq 'gettocart' || $switch eq 'close') {
+               } elsif (($switch =~ /^(add|addfromcart|get|gettocart)$/ && $items) || $switch eq 'close') {
                        if ($char->storage->isReady()) {
                                if ($switch eq 'add') {
                                        cmdStorage_add($items);
```

Comment (Mortimal) [reply src/Commands.pm]: k i understood il rewrite that

**Comment by Mortimal** [comment]
thx @allanon all fixed

File: src/Commands.pm [hunk 2]
```diff
@@ -4631,7 +4631,10 @@ sub cmdStorage {
 			cmdStorage_log();
 		} elsif ($switch eq 'desc') {
 			cmdStorage_desc($items);
-		} elsif ($switch eq 'add' || $switch eq 'addfromcart' || $switch eq 'get' || $switch eq 'gettocart' || $switch eq 'close') {
+		}elsif(($switch =~ /^(add|addfromcart|get|gettocart)$/ && $items)
```
Comment (allanon) [inline src/Commands.pm]: Sigh. Why make the whitespace ugly?

Comment (windhamwong) [reply src/Commands.pm]: lol I agree they are ugly, but at least it is working.

**Pull request merged**